src/components/weather/DailyWeather.tsx

```diff
@@ -52,20 +52,24 @@ export const DailyWeather = ({ date, description, icon, temperatures }: Props) =
           <RegularText />
           <WrapperRow spaceBetween>
             <View style={styles.dayTimeEntry}>
-              <RegularText>Morgens</RegularText>
-              <RegularText>{morn.toFixed(1)}°C</RegularText>
+              <RegularText center>
```

Comment:
why was `center` added? was ist aligned left with the new content?
and then the `styles.dayTimeEntry` of the parent view can be removed? or maybe the entire wrapping view, which was used to center?

Reply:
If I don't give RegularText the centre prop, the text is written left-aligned. I need to give the centre prop so that the design is the same as before.

Yes, you are right. We can remove the Views here and just use RegularText.